src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -69,41 +63,28 @@ const createTestBulkUpload = async (
 	});
 };
 
-const createTestBaseFields = async (): Promise<
-	[BaseField, BaseField, BaseField]
-> => {
-	const proposalSubmitterEmailBaseField = await createBaseField({
+const createTestBaseFields = async (): Promise<void> => {
```

Comment:
I probably wouldn't return void in situations like this, e.g. if we need the ids of the things created later, but the previous signature was a little wacky as well. :shrug: It's OK!